### PR TITLE
updating documentation

### PR DESCRIPTION
**Pull request by spunkedy**
Just detailing what I needed to do to get the documentation working

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,41 @@ If you need custom modifications to this system for your device, clone this
 repository and update as described in [Making custom
 systems](https://hexdocs.pm/nerves/customizing-systems.html).
 
+
+## Depedencies
+
+* OTP 25 or higher
+* libmnl-dev or equivilent
```
Comment (fhunleth) [inline README.md:48]: `libmnl-dev` shouldn't be needed. Did you get an error?

Comment (spunkedy) [reply README.md:48]: Yeah, in order to build the firmware I had to do it.

Comment (spunkedy) [reply README.md:48]: I believe it was around `nerves_system` if I remember correctly. I can try to re-clean and confirm

Comment (spunkedy) [reply README.md:48]: after cleaning it several times and removing, I can't replicate the error. will remove that for now.

**Comment by fhunleth** [comment]
Thanks!

**Pull request merged**